rasa/core/brokers/pika.py

```diff
@@ -276,12 +277,62 @@ def close(self) -> None:
 
     @property
     def rasa_environment(self) -> Optional[Text]:
+        """Get value of the `RASA_ENVIRONMENT` environment variable."""
         return os.environ.get("RASA_ENVIRONMENT")
 
+    @staticmethod
+    def _get_queues_from_args(
+        queue_arg: Union[List[Text], Tuple[Text], Text, None],
+        queues_arg: Union[List[Text], Tuple[Text], Text, None],
+    ) -> Union[List[Text], Tuple[Text]]:
+        """Get queues for this event broker.
```

Comment:
Let's add why we have to do this (e.g. for compatibility reasons, can be removed in the future, bla)